src/__init__.py

```diff
@@ -0,0 +1,12 @@
+import pkgutil
+from collections.abc import Iterator
+
+__version__ = "0.1.0"
+__all__ = ("get_package_names",)
+
+
+def get_package_names() -> Iterator[str]:
+    """Iterate names of all packages located in /src/."""
+    for package in pkgutil.iter_modules(__path__):
+        if package.ispkg:
+            yield package.name
```

Comment:
The folder name should probably be `sir-robin` (or `sir_robin`, maybe shortened to `sir_robin` depending on how Lancebot does it).